analyses/npl/04_glofas_flood_events.md

```diff
@@ -173,12 +119,13 @@ for istation, station in enumerate(FINAL_STATIONS):
 ```python
 # Print out some stats
 # Round to the nearest 5 for a presentation (10 seems a bit too coarse)
+rp = 1.5
 def round_to_5(x):
```

Comment:
Would this make sense to instead put as an option in ```utils.get_more_detection_stats()```?

Reply:
I put the rounding function in general utils, but I kept it out of `get_more_detection_stats` because I think you may want to keep the parameters there in fraction form for plotting (and can always round by hand later)